man/systemd-system.conf.xml

```diff
@@ -289,6 +289,20 @@
         <xi:include href="version-info.xml" xpointer="v239"/></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>ProtectSystem=</varname></term>
+
+        <listitem><para>Takes a boolean argument or the string <literal>auto</literal>. If set to true this
+        will remount <filename>/usr/</filename> read-only. If set to <literal>auto</literal> (the default)
+        and running in an initrd equivalent to true, otherwise false. This implements a restrictive subset of
```

Comment:
restrictive → restricted

"restrictive" sounds like an ever stronger variant of something, but actually it's a subset of the other thing.